src/dfmpreview/src/DfmProcess.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.IO;
+using System.Text;
+
+namespace Dfm_test
+{
+    using Microsoft.DocAsCode.Dfm;
+    using Microsoft.DocAsCode.MarkdownLite;
+    using Microsoft.DocAsCode.Plugins;
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
+
+                while (true)
+                {
+                    string num_str = Console.ReadLine();
+                    //Console.WriteLine(tmp + '\n');
+                    int num_of_row = System.Convert.ToInt32(num_str);
+                    //string str = "";
+                    StringBuilder str = new StringBuilder();
+                    //get the path;
+                    string path = Console.ReadLine();       //path -> basedir
+                    string filename = Console.ReadLine();   //filename -> the relative path of the current file 
+                    for (int i = 0; i < num_of_row; i++)
+                    {
+                        //tmp = Console.ReadLine();
+                        //Console.WriteLine(tmp + '\n');
+                        //str += Console.ReadLine();
+                        //str += "\r\n";
+                        str.Append(Console.ReadLine());
```

Comment:
fixed

---
In reply to: [73278056](https://github.com/dotnet/docfx/pull/558#discussion_r73278056) [](ancestors = 73278056)